llvm/lib/Transforms/Utils/LCSSA.cpp

```diff
@@ -74,21 +74,17 @@ static bool isExitBlock(BasicBlock *BB,
 /// For every instruction from the worklist, check to see if it has any uses
 /// that are outside the current loop.  If so, insert LCSSA PHI nodes and
 /// rewrite the uses.
-bool llvm::formLCSSAForInstructions(SmallVectorImpl<Instruction *> &Worklist,
-                                    const DominatorTree &DT, const LoopInfo &LI,
-                                    ScalarEvolution *SE,
-                                    SmallVectorImpl<PHINode *> *PHIsToRemove,
-                                    SmallVectorImpl<PHINode *> *InsertedPHIs) {
+static bool formLCSSAForInstructionsWorker(
```

Comment:
```suggestion
static bool formLCSSAForInstructionsImpl(
```
or alternatively give it the same name as the exported function and call it fully qualified with `::` prefix.

Reply:
Or XXXhelper -- but XXXImpl sounds good.

Reply:
went with Impl